src/games/supported/KeystoneKapers.hpp

```diff
@@ -0,0 +1,65 @@
+/* *****************************************************************************
+ * A.L.E (Arcade Learning Environment)
+ * Copyright (c) 2009-2013 by Yavar Naddaf, Joel Veness, Marc G. Bellemare and 
+ *   the Reinforcement Learning and Artificial Intelligence Laboratory
+ * Released under the GNU General Public License; see License.txt for details. 
+ *
+ * Based on: Stella  --  "An Atari 2600 VCS Emulator"
+ * Copyright (c) 1995-2007 by Bradford W. Mott and the Stella team
+ *
+ * *****************************************************************************
+ */
+#ifndef __KEYSTONEKAPERS_HPP__
+#define __KEYSTONEKAPERS_HPP__
+
+#include "../RomSettings.hpp"
+
+
+/* RL wrapper for KeystoneKapers */
+class KeystoneKapersSettings : public RomSettings {
+
+    public:
+        KeystoneKapersSettings();
+
+        // reset
+        void reset();
+
+        // is end of game
+        bool isTerminal() const;
+
+        // get the most recently observed reward
+        reward_t getReward() const;
+
+        // the rom-name
+		// MD5 be929419902e21bd7830a7a7d746195d
+        const char* rom() const { return "keystonekapers"; }

```

Comment:
To stay consistent with the naming scheme, I'll rename the rom keystone_kapers, & similarly for the other roms.